recipe/bld.bat

```diff
@@ -1,6 +1,9 @@
 REM Change Python header location.
 xcopy %LIBRARY_INC%\boost\python\python.hpp %LIBRARY_INC%\boost
 
+REM `cairo` was needed to generate high-quality PNGs for structure depiction,
+REM see https://www.rdkit.org/docs/Install.html?highlight=cairo#recommended-extras
+REM but we disable it intentionally staring from v2023.03.3
```

Comment:
Can you expand on the intention? For our future selves updating this feedstock.

Reply:
@M-Waszkiewicz-Anaconda Could you clarify the real reason for excluding `cairo`?

Reply:
that is snowflake requirement to remove  cairo-> libx11 dependency